### PR TITLE
send email for invitate to team

### PR DESCRIPTION
**Pull request by brusatimatias**

### CODE REVIEW

File: app/views/teams/_form.html.erb [hunk 1]
```diff
@@ -17,7 +17,10 @@
   </div>
 
   <div class="field">
-    <%= form.select :user_ids, options_from_collection_for_select(User.all, "id", "name"), { include_blank: true}, { multiple: true } %>
+    <%= form.select :user_ids, options_from_collection_for_select(
+        User.where.not("id = ?", current_user.id),"id", "name"),
```
Comment (mrplum) [inline app/views/teams/_form.html.erb]: Nice filter 👍. Just a note here. When company is integrated we should use `current_user.company.users` to bring only users of the current company

File: app/controllers/teams_controller.rb [hunk 2]
```diff
@@ -30,11 +30,15 @@ def edit
   # POST /teams
   # POST /teams.json
   def create
-    
+
     @team = Team.new(team_params)
 
     respond_to do |format|
       if @team.save
+        TeamUser.create(team_id: @team.id,user_id: current_user.id, joined: DateTime.now)
```
Comment (mrplum) [inline app/controllers/teams_controller.rb]: Here I think that invited user still didn't join to team. So I'm not sure about set `joined` attribute in this moment. wdyt?

File: config/routes.rb [hunk 3]
```diff
@@ -4,9 +4,10 @@
   resources :companies
 
   devise_for :users
-
   resources :users
-  resources :teams
+  resources :teams do
+    get 'accept_invitation', on: :collection
```
Comment (mrplum) [inline config/routes.rb]: here there is something weirdo. I think that it should be 
    
```  get 'accept_invitation', on: :member```

try to find here https://guides.rubyonrails.org/routing.html#adding-more-restful-actions about the difference between `member` and `collection`

File: app/controllers/teams_controller.rb (hunk 2 above)
Comment (brusatimatias) [reply app/controllers/teams_controller.rb]: seteo ese joined porque es cuando se agrega el creador del equipo(es para evitar tener que enviar una invitacion al creador)

File: config/routes.rb (hunk 3 above)
Comment (brusatimatias) [reply config/routes.rb]: lo hicimos con la coleccion para ir al buscar el TeamUser con esos dos parametros directamente

File: app/controllers/teams_controller.rb (hunk 2 above)
Comment (mrplum) [reply app/controllers/teams_controller.rb]: ahhh I see it now, thanks for explain, good catch 👍

**Pull request merged**